.storybook/test-storybook-canvas.config.js

```diff
@@ -17,6 +17,7 @@ const configuration = {
             'with North America viewport': DEFAULT_SCENARIO_CONFIG,
             'with World viewport': DEFAULT_SCENARIO_CONFIG,
             'with Include all data viewport': DEFAULT_SCENARIO_CONFIG,
+            'with location and hide zoom control button': DEFAULT_SCENARIO_CONFIG,
```

Comment:
what do you think about this?
`with disabled interactive and zoom control button`

Reply:
updated

Reply:
IMO, `with freezed map and hidden zoom control` :)